src/components/oppgaver/DokumentasjonKravView.tsx

```diff
@@ -91,6 +92,10 @@ const DokumentasjonKravView: React.FC<Props> = ({dokumentasjonkrav, dokumentasjo
 
     const [overMaksStorrelse, setOverMaksStorrelse] = useState(false);
 
+    const [filesHasErrors, setFilesHasErrors] = useState(false);
+
+    const [fileUploadingBackendFailed, setFileUploadingBackendFailed] = useState(false);
+
```

Comment:
Her ville jeg foreslått å gruble litt på å minimere states. All state man tilfører en komponent øker kompleksitet. For eksempel, er ikke "errorMessage" i praksis en funksjon av andre states?

For eksempel, hva om disse erstattes av en uploadStatus, som kan være typ `type UploadStatus = "success" | "uploading" | "tooLarge" | "backendError" | "fileError" | undefined`?

Da kunne man også definert noe a la `Record<UploadStatus, string | null> = {success: null, tooLarge: "vedlegg.ulovlig_storrelse_av_alle_valgte_filer"}` i toppen av filen for å si hvilke tilstander som skal ha feilmeldinger, og bruke error={errorMessages[uploadStatus]}.

Reply:
Er enig med at det blir for mye useState brukt. Foreløpig er det bare sånn i dokumentasjonkrav men det kommer også til bli seende sånn ut i dokumentasjonEtterspurt og i ettersendelse når de også får samme behandling som dokumentasjonkrav. 

Dette kan bli ordnet med neste refaktorering